example_src/devdemos/source_code_display.cljs

```diff
@@ -0,0 +1,103 @@
+(ns devdemos.source-code-display
```

Comment:
This should be a decent starting point for explaining what you can do with the changes I made. I'm not 100% clear on why cljs.repl sometimes cannot find the code in question, such as the example at the bottom.